docker/build-tools/Dockerfile

```diff
@@ -378,17 +379,17 @@ COPY --from=python_context /usr/local/bin /usr/local/bin
 COPY --from=python_context /usr/local/lib /usr/local/lib
 COPY --from=python_context /usr/local/google-cloud-sdk /usr/local/google-cloud-sdk
 
-RUN mkdir -p /home/gocache && \
-    mkdir -p /home/go && \
+RUN mkdir -p /go && \
+    mkdir -p /gocache && \
     mkdir -p /home/.kube && \
     mkdir -p /home/.config
 
 # TODO must sort out how to use uid mapping in docker so these don't need to be 777
 # They are created as root 755.  As a result they are not writeable, which fails in
 # the developer environment as a volume or bind mount inherits the permissions of
 # the directory mounted rather then overridding with the permission of the volume file.
-RUN chmod 777 /home/gocache && \
-    chmod 777 /home/go && \
+RUN chmod 777 /go && \
```

Comment:
`/home/cache` and
`/home/go` may remain in their current structure by using the rshared bindmount flag.  This may not be necessary or desireable - just pointing out the option is available.